tests/x11/x3270_ssl.pm

```diff
@@ -89,7 +88,13 @@ sub run {
     clear_console;
 
     type_string "cat $tracelog_file | tee /dev/$serialdev\n";
-    wait_serial "TLS/SSL tunneled connection complete", 5 || die "x3270 output doesn't match";
+
+    if ($current_ver >= 3.6) {
+        wait_serial "SSL_connect trace: SSLOK  SSL negotiation finished successfully", 5 || die "x3270 output doesn't match";
```

Comment:
Is this a typo SSLOK? No blank between SSL and OK?

Reply:
looks like not the typo,
It would be 
**SSL_connect trace: SSLOK**
plus
**SSL negotiation finished successfully**